### DISCUSSION TITLE
ConnectionError from my ScannerPro app - not valid WebDAV server

### DISCUSSION THREAD

**Discussion by stefancvetkovic** [jonaswinkler/paperless-ng#1061] [Support]
Hi,
I just wanted to check if I need to setup anything else for webdav protocol
Im getting this message from my scannerPro app : "not valid WebDAV server"
Im pretty sure that I put correct ip adress x.x.x.79:8000
Where I can find more about connecting webdav to paperless-ng? thanks

**Comment by jonaswinkler** [comment]
Paperless does not have WebDAV at all.

**Reply by stefancvetkovic** [reply]
But here I read that it does?
https://forums.unraid.net/topic/100843-support-paperless-ng-docker/

Number 3. in the first post..

**Reply by TheFehr** [reply]
They talk about a WebDav Server running on unraid. And the directory which is shared by that WebDav Server is the same one they mount into docker as the consumption directory.
